.config.template.toml

```diff
@@ -25,10 +25,17 @@ theme = { optional = false, default = "dark", example = "light", options = ["dar
     "light",
 ], explanation = "sets the Reddit theme, either LIGHT or DARK" }
 times_to_run = { optional = false, default = 1, example = 2, explanation = "used if you want to run multiple times. set to an int e.g. 4 or 29 or 1", type = "int", nmin = 1, oob_error = "It's very hard to run something less than once." }
-opacity = { optional = false, default = 0.9, example = 0.8, explanation = "Sets the opacity of the comments when overlayed over the background", type = "float", nmin = 0, nmax = 1, oob_error = "The opacity HAS to be between 0 and 1", input_error = "The opacity HAS to be a decimal number between 0 and 1" }
+opacity = { optional = false, default = 90, example = 80, explanation = "Sets the opacity (in percents) of the comments when overlayed over the background", type = "int", nmin = 10, nmax = 100, oob_error = "The opacity HAS to be between 10 and 100 percents", input_error = "The opacity HAS to be a number between 10 and 100" }
```

Comment:
Why did you have to change this

Reply:
I can revert it but I asked people around me and they said that percents is clearer than decimal number